### PR TITLE
feat: add initial samples

### PR DESCRIPTION
**Pull request by sofisl**

### CODE REVIEW

**Comment by sofisl** [comment]
@bcoe, I started the library off with Node 12 in order to not migrate it in the future. Should I also delete Node 10 configs g3?

File: samples/quickstart.js [hunk 1]
```diff
@@ -13,31 +13,23 @@
...
-  //TODO(library generator): write the actual function you will be testing
-  async function doSomething() {
-    console.log(
-      'DPE! Change this code so that it shows how to use the library! See comments below on structure.'
-    );
-    // const [thing] = await client.methodName({
-    // });
-    // console.info(thing);
+  async function listServices() {
+    const [thing] = await client.listServices({producerProjectId: projectId});
```
Comment (bcoe) [inline samples/quickstart.js:29]: perhaps rename `thing` to `services`? and sometimes if I know the response is a list, I might do:

```js
for (const service of services) {
  console.info(service);
}
```

File: samples/test/quickstart.js [hunk 2]
```diff
@@ -21,30 +21,25 @@ const path = require('path');
...
     projectId = await client.getProjectId();
   });
 
   it('should run quickstart', async () => {
-    //TODO: remove this line
-    // eslint-disable-next-line no-unused-vars
-    const stdout = execSync('node ./quickstart.js', {cwd});
-    //assert(stdout, stdout !== null);
+    const stdout = execSync(`node ./quickstart.js ${projectId}`, {cwd});
+    assert.match(stdout, /\[[\s\S]*\]/);
```
Comment (bcoe) [inline samples/test/quickstart.js:43]: we're testing that an empty array of services were returned? perhaps ignore my prior comment about printing each individual one 👍

**Review by bcoe — COMMENTED** [review]
smallest of nits.

File: samples/quickstart.js (hunk 1 above)
Comment (sofisl) [reply samples/quickstart.js:29]: Haha yes, sorry that was a vestige of the old code!

File: samples/test/quickstart.js (hunk 2 above)
Comment (sofisl) [reply samples/test/quickstart.js:43]: Yep, exactly!

**Review by bcoe — APPROVED** [review]

**Pull request merged**